SwiftCompilerSources/Sources/SIL/GlobalVariable.swift

```diff
@@ -82,6 +82,8 @@ extension Instruction {
         return type.isBuiltinInteger || type.isBuiltinFloat
       case .PtrToInt:
         return bi.operands[0].value is StringLiteralInst
+      case .IntToPtr:
```

Comment:
nitpick: The IntToPtr builtin stuff is worth putting into a separate commit, because it's really unrelated to the enum simplifications.